backend/prospect/views.py

```diff
@@ -192,13 +188,10 @@ def put(self, request, *args, **kwargs):
         response = requests.put(url, data=json.dumps(data))
 
         if response.status_code in [200, 201]:
-            response = response.json()
-            print(response)
-            if response["data"]["modified_documents"] == 0:
+            r = response.json()
+            if r["data"]["matched_documents"] == 0:
```

Comment:
Please check this out, last time I tested the Zuri Core API, it was `modified_documents` that counts, If it finds the document successfully, what happens if an error gets thrown that it couldn't modify it ??, let's say different data types?

Reply:
Thanks for the insight.

I have made changes to accounted for this case.

Reply:
Great, Looks Good to me 👍 .